app/templates/app/question_clue_spectrum.html

```diff
@@ -103,6 +120,32 @@ <h1>Waiting for other teammates to submit their clues....</h1>
     const submit_clue_1_btn = document.getElementById('submit_clue_1');
     const submit_clue_2_btn = document.getElementById('submit_clue_2');
     const please_wait = document.getElementById('please_wait');
+
+    $("#slider").roundSlider({
+            sliderType: "min-range",
+            radius: 130,
+            width: 16,
+            value: '{{generated_random_question_answer }}',
+            handleSize: 0,
+            handleShape: "square",
+            circleShape: "half-top",
+            min: 1,
+            max: 100,
+            disabled: true,
+    }); //end of .roundSlider 
+
+    $("#sliderTwo").roundSlider({

```

Comment:
What is the difference between the code above and the one below? consider adding a comment to differentiate

Reply:
added